deploy/etcd/docker-compose.yaml

```diff
@@ -0,0 +1,103 @@
+version: '3.8'
```

Comment:
This is good, and since it's for the test I won't even raise the question of why is this not autogenerated.

It should not be in top-level `deploy/` though, right? Since it clearly is for testing purposes only.

Reply:
yep, makes sense to move it somewhere closer to test folder